setup.cfg

```diff
@@ -39,6 +39,26 @@ wayland =
   # NOTE: When updating the major or minor version of pywlroots here, also update:
   # tox.ini, docs/manual/install/index.rst, docs/manual/wayland.rst
   pywlroots==0.16.4
+widgets =
+  dbus-next
+  imaplib2
+  iwlib
+  keyring
+  mailbox
+  psutil
+  pulsectl
+  pulsectl_asyncio
```

Comment:
I think `pulsectl_asyncio` pulls in `pulsectl`

Reply:
It does indeed

Reply:
We import pulsectl so figured it is a dep in its own right. Im not so familiar with the widget or those libs though so maybe they are always together?